operations/support-money.rst

```diff
@@ -0,0 +1,125 @@
+
+   Příspěvek na blogu je **podmínkou** jednorázových grantů. Články nám pomáhají získávat peníze od sponzorů a informují komunitu o tom, co děláme. Pokud použijete grant a článek nenapíšete, nedostanete další grant.
+
+   .. note::
+      Nemusíte válčit se systémem, na kterém blog běží. Článek napište v čem chcete, třeba v `Google Docs <https://docs.google.com/>`__, a lidi, kteří se kolem blogu motají, vám to do něj rádi pomohou vpravit.
+
+Příklad:
+
+   Zuzka by chtěla natisknout třička pro `PyLadies <https://pyladies.cz/>`__. Na tričkách nechce loga sponzorů, takže sponzory nehledá, požádá Pyvec o grant. Do `#money <https://pyvec.slack.com/messages/C9E81JFS5/>`__ napíše:
+
+   *@board Ahoj, jsem organizátorka PyLadies. Rádi bychom natiskli trička. Podle toho, jaký bude zájem, budeme potřebovat 10 až 30 000 Kč. Trička jsou jedinou odměnou dobrovolníkům, posilují soudržnost mezi organizátory i absolventkami kurzů a zároveň šíří povědomí o PyLadies, když v nich lidi chodí po světě.*
+
+   Výbor o tomto zahlasuje a odepíše, že grant bude přidělen. Následně Zuzka zjistí od lidí počty a velikosti triček, zadá zakázku firmě Tričkománie, s.r.o. a nechá u ní vystavit fakturu na Pyvec. Nakonec je to za 26 000 Kč. Protože jde o velkou částku, kterou Zuzka platit ze svého nechce, fakturu zaplatí přímo Pyvec. Když už je dost triček mezi lidma, Zuzka udělá pár fotek na mobil a napíše pár vět na `blog <https://blog.python.cz/>`__. Pro zajímavost přihodí i dvě čísla, kolik triček se vytisklo a jak rychle se rozdaly.
+
+
+.. _pausalni-granty:
+
+Paušální granty
+^^^^^^^^^^^^^^^
+
+Granty mohou být :ref:`jednorázové <granty>` nebo paušální. Ty paušální se hodí, pokud má nějaká skupina během své dlouhodobé činnosti více drobných výdajů. Příklad:
+
+   Lumír je hlavním organizátorem kurzů pro začátečníky v Zeleném Údolí. Kurzy se konají pravidelně každý půlrok a při jejich organizaci je potřeba tu koupit lístečky, tu prodlužovačku, tam zase malé občerstvení. Nejsou to velké výdaje, takže je Lumír platí sám, ale když se v prosinci podíval do prasátka, zjistil, že v součtu to vlastně bylo dost peněz. Rozhodne se tedy požádat Pyvec o paušální grant.
+
+Postup získání grantu je :ref:`stejný jako u jednorázových <granty>`, akorát s tím rozdílem, že nežádáte o jednorázové peníze, ale o pravidelnou částku na měsíc. Opět platí, že Pyvec může věci :ref:`proplácet <jak-proplatit>` jen oproti účetnímu dokladu, tzn. účtence nebo faktuře. Grant 200 Kč/měsíc, je tedy míněn "proplatíme účetní doklady do výše 200 Kč měsíčně". Po získání paušálního grantu **nemusíte psát článek na blog**. (Ale můžete!) Příklad:
+
+   Lumír do `#money <https://pyvec.slack.com/messages/C9E81JFS5/>`__ napíše:
+
+   *@board Zdar jak sviňa, v Zeleném Údolí děláme kurzy pro začátečníky, dva semestry ročně. Máme u toho malé výdaje, v podstatě kancelářské potřeby. Kurzy prošlo za poslední rok 40 lidí ročně a mají dlouhodobě skvělé ohlasy od účastníků. Pomohlo by nám proplatit 100 Kč měsíčně.*
```

Comment:
\<sarkasmus\>Nechceš tam dát rovnou třeba *Čuš pičus?*\</sarkasmus\>

Reply:
Už jsi druhý (Pražák), kterého to zaujalo 😄 Zvláštní! Mě to přijde jako úplně normální (brněnský?) pozdrav 🤔

Reply:
Teď mám strašnou chuť někam přidat "čauky mňauky".